src/pages/ImportData.js

```diff
@@ -299,8 +313,10 @@ const ImportData = () => {
           }],
           [Utils.DEFAULT, () => {
             return h(Fragment, [
-              h2({ style: styles.title }, ['Destination Workspace']),
+              h2({ style: styles.title }, ['Destination of the prepared data']),
               div({ style: { marginTop: '0.5rem' } }, ['Choose the option below that best suits your needs.']),
+              div({ style: { fontWeight: '600', marginTop: '1.5rem', fontSize: '1rem', lineHeight: '1.5' } }, ['Stay in Terra and combine it with other data you have and use Terra tools for analyses']),
```

Comment:
I think this prompt isn't needed until we have the cavatica export feature. Until then, it doesn't make sense to say `Stay in Terra` since there's no alternative offered.